CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Allow core-component configuration via `kubeadm --patches`
+- import the `cluster-shared` chart to apply additional resources in a Workload Cluster via `ClusterResourceSets`
+- renamed `labels.commonWithoutVersion` to `labels.selector` (to align with other providers)
```

Comment:
```suggestion
- Import the `cluster-shared` chart to apply additional resources in a Workload Cluster via `ClusterResourceSets`
- Renamed `labels.commonWithoutVersion` to `labels.selector` (to align with other providers)
```

Reply:
`Renamed` - `Rename`

It should also go to `### Changed`

Reply:
Correction:

```
- renamed `labels.commonWithoutVersion` to `labels.selector` (to align with other providers)
```

should be renamed from the changelog. This is only a refactoring if I'm not mistaken. For the end user it's transparent change.